app/src/main/java/eu/kanade/tachiyomi/util/CrashLogUtil.kt

```diff
@@ -22,13 +24,25 @@ class CrashLogUtil(private val context: Context) {
         try {
             val file = context.createFileInCacheDir("tachiyomi_crash_logs.txt")
             Runtime.getRuntime().exec("logcat *:E -d -f ${file.absolutePath}")
+            file.appendText(getDebugInfo())
 
             showNotification(file.getUriCompat(context))
         } catch (e: IOException) {
             context.toast("Failed to get logs")
         }
     }
-
+    fun getDebugInfo(): String {
+        return """
+            App version: ${BuildConfig.VERSION_NAME} (${BuildConfig.FLAVOR}, ${BuildConfig.COMMIT_SHA}, ${BuildConfig.VERSION_CODE}, ${BuildConfig.BUILD_TIME})
```

Comment:
Can you add an extra line break for this, so it can there can be a break in the txt between the logs and debug info

Reply:
```
--------- beginning of main
09-29 18:51:02.877  7642  7642 E chromium: [ERROR:filesystem_posix.cc(62)] mkdir /data/user/0/eu.kanade.tachiyomi.debugJ2K/cache/WebView/Crashpad: No such file or directory (2)
09-29 18:51:03.564  7642  7747 E FMQ     : grantorIdx must be less than 3
09-29 18:51:03.564  7642  7747 E FMQ     : grantorIdx must be less than 3
09-29 18:51:03.568  7642  7747 E ion     : ioctl c0044901 failed with code -1: Inappropriate ioctl for device
09-29 18:51:03.596  7642  7747 E FMQ     : grantorIdx must be less than 3
09-29 18:51:04.504  7642  7757 E SQLiteLog: (284) automatic index on C(manga_id)
09-29 18:51:04.504  7642  7757 E SQLiteLog: (284) automatic index on R(manga_id)
09-29 18:51:04.504  7642  7757 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
09-29 18:51:11.594  7642  7758 E SQLiteLog: (284) automatic index on mangas(_id)

App version: 1.3.1 (dev, be37ae1d8, 81, 2021-09-29T23:50:22.4261811)
Android version: 11 (SDK 30)
Android build ID: sdk_gphone_x86_arm-userdebug 11 RSR1.200819.001.A1 6823625 dev-keys
Device brand: google
Device manufacturer: Google
Device name: generic_x86_arm
Device model: sdk_gphone_x86_arm
Device product name: sdk_gphone_x86_arm
```

something like this? or did you want something like `--------- beginning of main` separation?

Reply:
Just this is enough, no need to add the same kind of separator as that. I thought about this debug info can be above the logs instead but I might not look at this info often so it's good as is